siteapp/views.py

```diff
@@ -1370,15 +1370,25 @@ def move_project(request, project_id):
     request ([HttpRequest]): The network request
     project_id ([int|str]): The id of the project
     Returns:
-        [JsonResponse]: Either a ok status or an error
+        [JsonResponse]: Either an ok status or an error
     """
-    try:
-        new_portfolio_id = request.POST.get("new_portfolio", "").strip() or None
+      try:
+        new_portfolio_id = request.POST.get("new_portfolio", "").strip()
         project = get_object_or_404(Project, id=int(project_id))
         cur_portfolio = project.portfolio
         new_portfolio = get_object_or_404(Portfolio, id=int(new_portfolio_id))
+    except:
+        return JsonResponse({"status": "error", "message": "Portfolio entered does not exist."})
+
+    # Check if the user moving the project is a superuser or
+    # if they are the owner of the project and have edit permissions in the target directory
+    owner = True if request.user.has_perm('can_grant_portfolio_owner_permission', cur_portfolio) else False
+    if request.user.is_superuser or ((request.user in project.get_admins()) or owner and 'change_portfolio' in get_perms(request.user, new_portfolio)):
```

Comment:
```suggestion
    if request.user.is_superuser or (request.user in project.get_admins() or owner and 'change_portfolio' in get_perms(request.user, new_portfolio)):
```
There was an extra set of parentheses around project.getadmins(). Removed the parentheses.